pyproject.toml

```diff
@@ -84,7 +84,7 @@ dev = [
     "invoke-kubesae>=0.1.0",
     "ipython>=8.32.0",
     "isort>=6.0.0",
-    "kubernetes==32.0.0",
+    "kubernetes~=32.0.1",
```

Comment:
Does `uv` allow for pinning only the major version? That way we'd pin only `32` and let it figure out the `.x.x`?

Reply:
I think that's what the [compatible release version specifier](https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release) is doing here. This is basically saying `32.0.*` to pick up any bug fix releases.

Reply:
Ohh, I misinterpreted. Updated!